versions.json

```diff
@@ -1,4 +1,5 @@
 {
+    "sketcher": "2026.1.0",
```

Comment:
Should we put this in a different file?  Adding it here means that we're going to need to rebuild all of the dependencies whenever the Sketcher version changes because of the `hashFiles('versions.json')` in the dependency `cache-key`.  One option would be to leave `external/versions.json` as is and add a separate file in the repo root directory with the `sketcher` version.